pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -163,5 +163,8 @@ func (c *CloudProvider) ValidateSpec(ctx context.Context, spec *v1alpha3.Provisi
 	if spec.Cluster.Name == nil || len(*spec.Cluster.Name) == 0 {
 		errs = errs.Also(apis.ErrMissingField("name")).ViaField("cluster")
 	}
+	if spec.Cluster.CABundle == nil || len(*spec.Cluster.CABundle) == 0 {
```

Comment:
check out `ptr.StringValue(spec.Cluster.CaBundle) == ""`, which handles both of these cases.

Does this mean this can no longer be nil?

Reply:
Ok!

> Does this mean this can no longer be nil?

For AWS, it has to either be specified or else in `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`.

Reply:
Doesn't this validation prevent the case where I set the CABundle to nil (i.e., so that it falls back to the ca.cert file).

Reply:
I added a test for that - see line 591 in `suite_test.go`. AFAICT it's better for most of the code to run w/ defaulting and then explicitly turn it off as needed.

Reply:
FYI, latest code has the `ptr.StringValue` usage